static/usr/lib/snapd/sshd-host-keygen

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
```

Comment:
This unit needs to be documented. It should say how the unit relates to ssh configuration.